src/backend/aspen/config/config.py

```diff
@@ -303,3 +310,38 @@ def NEXTSTRAIN_SPOT_MEMORY(self) -> str:
     @property
     def NEXTSTRAIN_SPOT_VCPU(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["RunSPOTVcpu"]
+
+    ####################################################################################
+    # Pangolin SFN config properties
+    @property
+    def PANGOLIN_DOCKER_IMAGE_ID(self) -> str:
+        print(self.AWS_PANGOLIN_SFN_PARAMETER)
+        return self.AWS_PANGOLIN_SFN_PARAMETER["Input"]["Run"]["docker_image_id"]
+
+    @property
+    def PANGOLIN_SFN_ARN(self) -> str:
+        return self.AWS_PANGOLIN_SFN_PARAMETER["SFNArn"]
+
+    @property
+    def PANGOLIN_OUTPUT_PREFIX(self) -> str:
+        return self.AWS_PANGOLIN_SFN_PARAMETER["OutputPrefix"]
+
+    @property
+    def PANGOLIN_WDL_URI(self) -> str:
+        return self.AWS_PANGOLIN_SFN_PARAMETER["RUN_WDL_URI"]
+
+    @property
+    def PANGOLIN_EC2_MEMORY(self) -> str:
```

Comment:
Any thoughts on whether we should break all these parameters into individual settings vs just having general "pangolin_job_params" and "nextstrain_job_params" type settings that contain the necessary info?

Reply:
I can't really think of a great approach here right now, but I'm certainly not attached to this one. It makes it easy to see what's available at a glance but anyone tinkering with this could easily look it up. There's no reason they couldn't just be used by accessing config.AWS_PANGOLIN_SFN_PARAMETER with the right key

Reply:
Alright, just curious. I think we can revisit this in the future. Excited to see this feature working!

Reply:
I ended up getting rid of them, I realized that where I am using them I am constructing a JSON string with the same keys as the ones I'm accessing, so no memorization even needed - it's pretty natural.